src/main/kotlin/graphics/scenery/primitives/Atmosphere.kt

```diff
@@ -12,56 +12,73 @@ import java.lang.Math.toRadians
 import java.time.LocalDateTime
 import kotlin.collections.HashMap
 import kotlin.math.*
+import kotlin.time.Duration.Companion.seconds
 
 /**
  * Implementation of a Nishita sky shader, applied to an [Icosphere] that wraps around the scene as a skybox.
  * The shader code is ported from Rye Terrells [repository](https://github.com/wwwtyro/glsl-atmosphere).
  * To move the sun with arrow keybinds, attach the behaviours using the [attachRotateBehaviours] function.
- * @param initSunDir [Vector3f] of the sun position. Defaults to sun elevation of the current local time.
- * @param emissionStrength Emission strength of the atmosphere shader. Defaults to 0.3f.
+ * @param initSunDirection [Vector3f] of the sun position. Defaults to sun elevation of the current local time.
```

Comment:
And what happens if you leave it at `null`, as per the default?

Reply:
Also, just fyi, for KDoc, you don't need to use the `@param` annotation from Javadoc, but you can just refer to the parameters as e.g. `[initSunDirection]`.